docs/en/query_language/table_functions/odbc.md

```diff
@@ -0,0 +1,27 @@
+# odbc {#table_functions-odbc}
+
+Returns table that is connected via ODBC driver.
```

Comment:
Provides access to table in external database via ODBC (with link to wikipedia again).
Mention driver for the first time later, near bridge.